src/components/shared/CardList/Card/Card.jsx

```diff
@@ -26,7 +26,9 @@ export class Card extends React.Component {
     }
 
     render() {
-        const src = this.props.card.cardId ? require(`../../../../images/cards/card (${this.props.card.cardId}).jpg`) : require('../../../../images/cardBack.jpg');
+        let src;        
+        if (this.props.gameMode === 'telltales') src = this.props.card.cardId ? require(`../../../../images/cards/card (${this.props.card.cardId}).jpg`) : require('../../../../images/cardBack.jpg')
+        else if (this.props.gameMode === 'custom') src=this.props.card.url;        
```

Comment:
might be nicer as a switch statement given there's defined options but that's down to your opinion?

Reply:
I would suggest multi lining the ternaries so it's a bit easier to follow like:
statement
   ? option 1
   : option 2

Reply:
what did you have in mind?

Reply:
a switch on the game mode or a switch on the cardId

Reply:
Or:
```js
const src = {
  telltales: this.props.card.cardId ? require(`../../../../images/cards/card (${this.props.card.cardId}).jpg`) : require('../../../../images/cardBack.jpg'),
  custom: src=this.props.card.url
}[this.props.gameMode]
```

Reply:
something like 
```
switch(this.props.gameMode) {
    case "telltales" 
         ternary...
         break
    case "custom"
         other thing...
         break
}
```
But Ronan's thing is also cool, it was just a clarity thing since you had nested ifs and ternaries

Reply:
I only suggested that other option because it's something we do quite a lot on my project.